plugins/notifications-backend/src/service/router-test-ts

```diff
@@ -0,0 +1,53 @@
+import { getVoidLogger } from '@backstage/backend-common';
```

Comment:
Is this file the right name?

Reply:
nope, committed by mistake, will be removed

Reply:
done